pkg/migrations/internal/seq_num.go

```diff
@@ -4,7 +4,7 @@ var (
 	// CurrentDBVersionSeqNum is the current DB version number.
 	// This must be incremented every time we write a migration.
 	// It is a shared constant between central and the migrator binary.
-	CurrentDBVersionSeqNum = 180
+	CurrentDBVersionSeqNum = 181
```

Comment:
Bump `MinimumSupportedDBVersionSeqNum` as well.

Reply:
do we have a check for this? Maybe a small action that will verify all numbers and fails on duplicates, skips etc.